.pyup.yml

```diff
@@ -1,4 +1,7 @@
-# autogenerated pyup.io config file 
+# autogenerated pyup.io config file
 # see https://pyup.io/docs/configuration/ for all available options
 
 update: insecure
+requirements:
+  - Pipfile
```

Comment:
We should consider adding Pipfile.lock, as that's currently used for the installation with Servicebook.

Reply:
Added!